pkgs/development/python-modules/boto3/default.nix

```diff
@@ -13,11 +13,11 @@
 
 buildPythonPackage rec {
   pname =  "boto3";
-  version = "1.9.96"; # N.B: if you change this, change botocore too
+  version = "1.9.152"; # N.B: if you change this, change botocore too
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c103241394d396ee08548b03d5d1f0f89a7ad1dfa7ccca88a47131f329cca093";
+    sha256 = "14x0rhlz6cbilj4cmk57vz8pm6pqyb8xa0wbah4fx5nkjh9j80y1";
```

Comment:
this is the wrong hash,

```
diff --git a/pkgs/development/python-modules/boto3/default.nix b/pkgs/development/python-modules/boto3/default.nix
index 0134bee194a..c5d51e56fc6 100644
--- a/pkgs/development/python-modules/boto3/default.nix
+++ b/pkgs/development/python-modules/boto3/default.nix
@@ -17,7 +17,7 @@ buildPythonPackage rec {

   src = fetchPypi {
     inherit pname version;
-    sha256 = "14x0rhlz6cbilj4cmk57vz8pm6pqyb8xa0wbah4fx5nkjh9j80y1";
+    sha256 = "0qspp97rddshjfqmmnqbvrszr66wq0ag05j8lcfsb9rj6ymz5s5s";
   };

   propagatedBuildInputs = [ botocore jmespath s3transfer ] ++ lib.optionals (!isPy3k) [ futures ];
```

Reply:
Fixed, seems like i used some other version hash when i was figuring out the relation between the cli boto3 and botocore